README.md

```diff
@@ -1,2 +1,230 @@
-# terraref-datasets
-Repository for code and small datasets derived from the TERRA REF program
+# [TERRA-REF Data](https://www.terraref.org/)
+
+This repository contains source code for accessing and curating [TERRA-REF](https://terraref.ncsa.illinois.edu/bety/) data used to support the [GenoPhenoEnvo](https://github.com/genophenoenvo) project and machine learning research. This repository supports one of our goals to provide open data and reproducible code in order to follow [FAIR](https://www.go-fair.org/fair-principles/) data principles and contribute to open science.
+
+This repository focuses on _Sorghum bicolor_ trait data collected from four experiments and the associated weather data for those locations, listed below.
+
+- Maricopa Agricultural Center, University of Arizona, Season 4
+    - Coordinates: 33.069, -111.972
+    - Elevation: 362 meters
+    - Planting: 2017-04-20, Day 110
+    - Last Day of Harvest: 2017-09-16, Day 259
+- Maricopa Agricultural Center, University of Arizona, Season 6
+    - Coordinates: 33.068941, -111.972244
+    - Elevation: 362 meters
+    - Planting: 2018-04-25, Day 115
+    - Harvest: 2018-08-01, Day 213
+- Kansas State University, Ashland Bottoms
+    - Coordinates: 39.126, -96.677
+    - Elevation: 325 meters
+    - Planting: 2016-06-17, Day 169
+    - Harvest: 2016-10-21, Day 295
+- Clemson University Pee Dee Research and Education Center, South Carolina
+    - Coordinates: 34.289, -79.737 
+    - Elevation: 42 meters
+    - Planting: 2014-05-06, Day 126
+    - Latest date in Clemson trait data: 2014-10-15, Day 288
+    
+### Licenses
+TERRA-REF data and other documents are licensed under CC-By. All software is licensed under MIT.
+
+### Contacts
+Please contact these individuals with any questions
```

Comment:
Remove Todd and Nadia's name from this derivative dataset that they are not familiar with. 

Also, add an empty line below the heading `### Contacts`